commands/signout.js

```diff
@@ -21,8 +21,11 @@ async function signout (argv) {
 function printHelp () {
   helpHeader(
     'signout',
-    chalk`ncm {${COLORS.yellow} signout}`,
-    'ncm signout'
+    chalk`{${COLORS.light1} ncm} {${COLORS.yellow} signout}`,
+    'ncm signout',
+    `
+Reset login tokens (sign out).
```

Comment:
```suggestion
Sign out.
```

Resetting tokens is an implementation detail.